libdevice/device_itt.h

```diff
@@ -0,0 +1,58 @@
+//==------- device_itt.h - ITT devicelib functions declarations ------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//==------------------------------------------------------------------------==//
+
+#ifndef __LIBDEVICE_DEVICE_ITT_H__
+#define __LIBDEVICE_DEVICE_ITT_H__
+
+#include "device.h"
+
+#ifdef __SPIR__
+#include <cstddef>
+#include <cstdint>
+
+// Use SPIRV constants directly in place of OCL intrinsic functions.
+#define __SPIRV_VAR_QUALIFIERS EXTERN_C const
+typedef size_t size_t_vec __attribute__((ext_vector_type(3)));
```

Comment:
Hi, @vzakhari 
It looks like those spirv builtins have been declared in spirv_vars.hpp, what about including spriv_vars.hpp here? "assert" impl also uses the spriv builtin to get gid, wid: https://github.com/intel/llvm/blob/sycl/libdevice/crt_wrapper.cpp#L36

Reply:
There is no intersection between `SPIR-V` variables used for `crt` and `itt`, but I guess it is okay to put it all into `spir_vars.h`.  I will do it.